lib/content-services/src/lib/content-node-share/content-node-share.dialog.spec.ts

```diff
@@ -137,7 +137,7 @@ describe('ShareDialogComponent', () => {
         expect(sharedLinksApiService.createSharedLinks).toHaveBeenCalled();
         expect(renditionService.getNodeRendition).toHaveBeenCalled();
         expect(fixture.nativeElement.querySelector('input[formcontrolname="sharedUrl"]').value).toBe('some-url/sharedId');
-        expect(fixture.nativeElement.querySelector('.mat-slide-toggle').classList).toContain('mat-checked');
+        expect(fixture.nativeElement.querySelector('[data-automation-id="adf-share-toggle"]').classList).toContain('mat-checked');
```

Comment:
This: 
fixture.nativeElement.querySelector('[data-automation-id="adf-share-toggle"]').classList

is repeated couple times. Can you move that line to function and reuse?